webapp/web-components/cht-form/src/stubs/language.service.ts

```diff
@@ -0,0 +1,11 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class LanguageService {
```

Comment:
This is only here so this code works in the config test harness?

Reply:
The real implementation of the `LanguageService` pulls in a bunch of other services that it depends on (specifically, I think the changes.service does not work in cht-form and keeps spewing errors into the console logs).  This change just isolates cht-form from the entire LanguageService dependency tree.  So, this is not just for the test-harness, but is required generally for proper operation of cht-form.

Reply:
> I think this in-concrete coupling between cht-form and other webapp services confuses me

Yeah, not just you.  :sweat:   Conceptually we are just generating two separate artifacts (an Angular `webapp` and a generic `cht-form` web component) from basically the same code base. But, that does mean that changes to the codebase can affect both artifacts.  Frustratingly, I have not been able to find much in the way of Angular tooling to help manage/detect changes in the service dependency hierarchy. It would be nice to at least get a clear notification when new service dependencies end up pulled in to cht-form.

So far, the best we have is trying to keep the unit/integration tests tight so we know exactly when stuff breaks...